Python/Product/TestAdapter.Executor/TestAdapter.Executor.csproj

```diff
@@ -93,21 +93,19 @@
     <Compile Include="Common\Config\RunSettingsUtil.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="Pytest\PytestExtensions.cs" />
-    <Compile Include="Pytest\TestDiscovererPytest.cs" />
-    <Compile Include="Pytest\TestExecutorPytest.cs" />
-    <Compile Include="Common\Services\DiscovererFactory.cs" />
+    <Compile Include="Pytest\PytestTestDiscoverer.cs" />
+    <Compile Include="Pytest\PytestTestExecutor.cs" />
     <Compile Include="Common\Services\ExecutorService.cs" />
     <Compile Include="Pytest\JunitXmlTestResultParser.cs" />
-    <Compile Include="Common\Services\CodeCoverage.cs" />
-    <Compile Include="Common\Services\IPythonTestDiscoverer.cs" />
+    <Compile Include="Common\Services\CodeCoverage.cs" />   
     <Compile Include="Common\Services\ProcessExecute.cs" />
     <Compile Include="Common\Services\PythonDebugMode.cs" />
     <Compile Include="Common\TestCollection.cs" />
     <Compile Include="Common\PythonTestDiscoverer.cs" />
     <Compile Include="Common\TestProtocol.cs" />
     <Compile Include="Common\TestReader.cs" />
-    <Compile Include="UnitTest\TestExecutorUnitTest.cs" />
-    <Compile Include="UnitTest\TestDiscovererUnitTest.cs" />
+    <Compile Include="UnitTest\UnitTestTestExecutor.cs" />
```

Comment:
Same here, file was renamed but not the class

Reply:
thanks